benchmarks/u_rho/benchmark_output/README

```diff
@@ -0,0 +1 @@
+This directory contains benchmark outputs
```

Comment:
In principle we don't need this file. Snakemake will create all the directories.

Reply:
Deleted!